microsoft-365/admin/setup/domains-faq.yml

```diff
@@ -233,3 +233,19 @@ sections:
             
           Want to know more about becoming the admin for your school? [Learn all about it](/microsoft-365/education/deploy/becoming-an-admin-in-office-365-education
           ).
+
+      - question: How do I create a website with Microsoft 365?
+        answer: |
+          Microsoft 365 does not include a public website for your business. To set one up, consider using a web hosting company like GoDaddy, Wix, or WordPress.
+
+          > [!VIDEO https://www.microsoft.com/videoplayer/embed/4839abc6-9323-4cbf-a79d-2907235f9ebb?autoplay=false]
+
+          1. Search for a web hosting provider where you can build your website.
+              You may find it easiest to choose the website builder at the same company where your domain name is registered.
+          
+          1. Design your business website, making sure to include your business logo, business purpose, text, images, and links.
+          
+          1. To integrate the website with Microsoft 365, you can add your business email to the website and a link to Microsoft Teams, SharePoint, or other collaboration tool.
+          
+          1. Make sure to connect your website to your domain name, which you&#39;re also using for email on Microsoft 365.
```

Comment:
```suggestion
          1. Make sure to connect your website to your domain name that you're also using for email on Microsoft 365.
```

Reply:
As per @pebaum request, please don't merge this PR. Thank you.